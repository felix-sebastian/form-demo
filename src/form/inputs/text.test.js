import React from "react";
import Text from "./text";
import renderer from "react-test-renderer";

it("displays the current value", () => {
  const result = renderer.create(<Text value={"Foo"} />).toJSON();
  expect(result).toMatchSnapshot();
});
