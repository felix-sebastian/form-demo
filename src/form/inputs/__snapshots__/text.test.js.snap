// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the current value 1`] = `
<input
  className="sc-AykKC bfeSMK"
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  value="Foo"
/>
`;
